src/main/containers/SourcePanel.jsx

```diff
@@ -1,18 +1,19 @@
 import React, { useState } from 'react';
+import DataCanvas from './DataCanvas';
```

Comment:
for consistency and the linter, we should append the .jsx file type when importing.

Reply:
I’m confused by this—I’ve experienced errors before when using the extension… whereas when I omit it, things seem to work and I’m not _exactly_ sure why…

Reply:
Scratch that, must’ve been a fluke. Will add in to all imports.